### ISSUE TITLE
Problems with 4 display and on/off extention

### ISSUE THREAD

**Issue by eyakubovskiy** [micheleg/dash-to-dock#794]
Hello.
1) The application "Dash to dock" contains a problem. I have 4 displays. Often I can not move the cursor from the top monitors to the bottom ones. The cursor rests on an invisible border. It turns out to go only in the crosshairs of the displays.
2) Disabling the extension does not work in the extension settings. If the extension is disabled, the settings are applied. Disabling / enabling does not change the display dock.
Sorry for my bad english.
Thank you!

v63
Linux Mint 19 + Gnome 3 Ubuntu

Best regards, Evgeniy

**Comment by micheleg** [comment]
Is it something specific to having 4 displays? Is is due to having tow displays positioned on on top of the other? Could you please test this?

Alos, please reports your settings (autohide? dock position?)

**Comment by eyakubovskiy** [comment]
No. The situation is when there are two displays too (top and bottom).
Settings: from below, on all monitors, monitors are isolated, not along the entire length, autohide, pressure 170.
Today I tested the situation. I set the pressure to "1" and the problem disappeared. Probably this is not a bug, but a feature of the installed settings. Sorry.
Thank you.

**Comment by eyakubovskiy** [comment]
Close issue?

**Comment by micheleg** [comment]
If you can move beyond the  "invisible border"  just by "pushing  harder", than it's definitely the intended behaviour. If instead the mouse is stack on the screen regardless of how strong you push, then it is a bug.

**Comment by eyakubovskiy** [comment]
In this case, this is a bug.
Exactly disappears if I turn off "pushing harder".
